src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" data-qa="big">
+      <img
+        src="images/Search.svg"
+        alt="image"
+        class="bigSearch-svg"
+      >
+      <input
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
      >
      
      <input
```